examples/concepts/javabip/casinoBroken/Casino.java

```diff
@@ -84,7 +96,7 @@ public void receiveBet(@Data(name = PLAYER) Integer sender,
     }
 
     @Transition(name = CASINO_WIN, source = BET_PLACED, target = IDLE, guard = "IS_OPERATOR & !GUESSED")
-    public void casinoWin(@Data(name = OPERATOR) Integer sender) {
+    public void casinoWin(/*@Data(name = OPERATOR) Integer sender*/) {
```

Comment:
Is this an actual technically needed change, or are the arguments just not necessary here? (As they are not used I would agree they seem useless)

Reply:
Ah, I guess the IDE gives a warning or compile error if arguments are removed. Best to remove them, then.

Reply:
No, wait, it is used: by the IS_OPERATOR guard, right? Then we should not remove it.